### ISSUE TITLE
Exit code is not non-zero when running as non-root

### ISSUE THREAD

**Issue by gtkramer** [clearlinux/clr-installer#687]
**Describe the bug**
clr-installer requires root privileges to perform an install.  However, when run as a regular user, clr-installer notifies the user of this but then exits cleanly with a code of 0.

**To Reproduce**
As a non-root user, run the following:
```
$ clr-installer --config <any config>
clr-installer MUST run as 'root' user to install! (user=1000)
```
The exit code is zero.

**Expected behavior**
When running the above as a non-root zero, the exit code should be non-zero.

**Screenshots**
N/A

**Environment (please complete the following information):**
 - Clear Linux OS Version: 32670
 - Installer Version 2.4.2
 - Platform: NUC Skykale

**Additional context**
N/A

**Comment by lebensterben** [comment]
It should exit with 126 for permission related problem.

**Comment by mdhorn** [comment]
@gtkramer This was by design originally to ensure the code followed the standard existing flow, but without the stack trace. I'm open to changing it.

**Comment by mdhorn** [comment]
Released in [2.4.3](https://github.com/clearlinux/clr-installer/releases/tag/2.4.3) in Clear Linux OS 32750 or higher.
